### ISSUE TITLE
train_diffusion_2d.py 

### ISSUE THREAD

**Issue by luhaihenchang** [xmed-lab/GenericSSL#5]
Hi, thanks for the code. I would like to ask, do train_diffusion_2d.py and train_diffusion.py in your code handle 2d slice data and 3d data respectively?
Is there a big difference in accuracy between running train_diffusion_2d.py and train_diffusion.py?

**Comment by McGregorWwww** [comment]
Thanks for your attention. 

Yes, the two training scripts are independent. Actually, the 2D pipeline was included as a supplement to enhance the codebase's versatility for 2D tasks, we did not make a thorough comparison between 2d and 3d. 
For the datasets mentioned in the paper, using 3D pipeline is better generally.  

Hope this helps.
